src/h5web/vis-packs/core/complex/MappedComplexLineVis.tsx

```diff
@@ -0,0 +1,36 @@
+import type { DimensionMapping } from '../../../dimension-mapper/models';
+import shallow from 'zustand/shallow';
+import type { H5WebComplex } from '../../../providers/models';
+import { usePhaseAmplitudeValues } from './hooks';
+import { useComplexConfig } from './config';
+import MappedLineVis from '../line/MappedLineVis';
+import { ComplexVisType } from './models';
+
+interface Props {
+  value: H5WebComplex[];
+  dims: number[];
+  dimMapping: DimensionMapping;
+  title: string;
+}
```

Comment:
For now, I declare only the props that are used in `ComplexLineContainer`, I will add the remaining props once I add NeXus support.